import os
import sys
from CeleryPy import log
from DB import DB
import Capture 
import Image
x=str(DB.getcoordinates())
log('ya', message_type='error', title='FUNCO')

